packages/batch-poster-monitor/index.ts

```diff
@@ -348,46 +397,54 @@ const monitorBatchPoster = async (childChainInformation: ChainInfo) => {
   // Get the last block of the chain
   const latestChildChainBlockNumber = await childChainClient.getBlockNumber()
 
-  if (!sequencerInboxLogs || sequencerInboxLogs.length === 0) {
-    // No SequencerInboxLog in the last 24 hours (time hardcoded in getDefaultBlockRange)
-    // We compare the "latest" and "safe" blocks, and check if any of the pending blocks contain any user-transactions
-    // NOTE: another way of verifying this might be to check the timestamp of the last block in the childChain chain to verify more or less if it should have been posted
-    const latestChildChainSafeBlock = await childChainClient.getBlock({
-      blockTag: 'safe',
-    })
+  const latestChildChainSafeBlock = await childChainClient.getBlock({
```

Comment:
I think most of this code can be inside the `if (!sequencerInboxLogs...)`. That way we can save multiple RPC calls when there are logs (which hopefully will be the common case)

Reply:
Updated